test/unit/constants.test.ts

```diff
@@ -0,0 +1,37 @@
+describe("BASE URL when", () => {
+	// Mock the process.env
+	beforeEach(() => {
+		jest.resetModules(); // Reset modules to clear cached values
+	});
+
+	describe("NODE_ENV = production", () => {
+		it("should return the production host URL", () => {
+			process.env.NODE_ENV = "production";
+
+			const BASE_URL = require("../../src/utils/constants").BASE_URL;
```

Comment:
Does
```suggestion
			const BASE_URL = require("@utils/constants").BASE_URL;
```

Not work? I'm assuming the transpiled JS doesn't support path shortcuts?

Reply:
Oh it works... tried once before and it was breaking, dk why. changing to @utils/constants